Markdown/Markdown.sublime-syntax

```diff
@@ -643,13 +643,18 @@ contexts:
     - include: inline-bold-italic-linebreak
     - include: scope:text.html.basic
   raw:
-    - match: '(`+)'
-      scope: punctuation.definition.raw.begin.markdown
-      push:
-        - meta_scope: markup.raw.inline.markdown
-        - match: '\1'
-          scope: punctuation.definition.raw.end.markdown
-          pop: true
+    - match: '{{backticks}}'
```

Comment:
Actually, I just realized that this is problematic in case a backtick is used but a raw block is not intended. However, I do believe that it's better to call people out on their backticks and require them to escape them (`` \` ``) than not supporting multi-line inline raw segments.